packages/web/pages/player/[username].tsx

```diff
@@ -300,7 +300,7 @@ export const getStaticPaths: GetStaticPaths<QueryParams> = async () => {
     paths: playerUsernames.map((username) => ({
       params: { username },
     })),
-    fallback: true,
+    fallback: false,
```

Comment:
here as well, fallback should be true

Reply:
We should deal with these cases as described in https://nextjs.org/docs/basic-features/data-fetching#fallback-true

Reply:
If we add fallback:true, the page tries to render with header, but since there is no data, it immediately turns back to 404 page which doesnt have any header. I think that is not a pretty UX. If fallback is not true like here, the page doesnt force itself for showing the content, if it is not available, it renders the 404 page without a hassle.

Reply:
@firattale fallback: true is necessary in case new players login for the first time and their profile page is not rendered yet on the server.

Reply:
I think fallback:blocking may help here as it is working similar to true but doesnt fallback to 404.

`fallback: 'blocking'

This is quite similar to fallback: true, except that it does not return the dummy loading page when a page that hasn't been cached is hit for the first time

Instead, it just makes the browser hang, until the page is rendered for the first time.

Future requests to that page are quickly served from the cache however, just like fallback: true.

 `

Reply:
wdty? @dan13ram

Reply:
Sounds great to me. But I just hope the blocking is not too long. Otherwise, it makes sense to display a loader to the user.